### ISSUE TITLE
Files seem to be corrupt

### ISSUE THREAD

**Issue by BlackStar-EoP** [drkmsmithjr/NixiePowerSupply#1]
Zip files with Gerber files cannot be opened, with KiCad, all files give error messages.

**Comment by drkmsmithjr** [comment]
Hi BlackStar-EoP.  I downloaded your fork and was not able to reproduce your error.  Both power power supply gerber files were able to be read by the Kicad GerbView.   For example, for this file below.  Please unzip all the files,  then start up GerbView from within the Kicad Tools area (CNTL-G).   Go to open the files and select all the files together.  They should load.    As you may know, the Gerber Files are what are needed for manufacturing.   

The PCB and Schematic files were written in Kicad 4.0 and some conversion may be required to run with the latest version of Kicad (5.0).  If you are having issues with this I can work to load a version 5.0.  Let me know.

thanks.     





[NixieSupply5vto160vDCMBoost_Rev3.zip](https://github.com/drkmsmithjr/NixiePowerSupply/files/7740323/NixieSupply5vto160vDCMBoost_Rev3.zip)

**Comment by drkmsmithjr** [comment]
Kicad version 5.0 have been updated to Git.

**Comment by BlackStar-EoP** [comment]
Hey drkmsmithjr, I have now tried ZIP files from both my fork and your original ones. They seem to be working now. I am unsure what has happened here to be honest. Basically, every file I downloaded was corrupt (diptrace couldn't even open the loose gerber files), the zip files couldn't be opened, but I can't reproduce that right now.

I do have the corrupt files though, NixieSupply5vto160vDCMBoost_Rev3.zip is 160kB while the real version is 32kB. Clearly something went wrong here. Was downloaded via browser, not via git clone.

I'd say this issue can be closed :)
